scala-xml/src/main/scala/scalaxml/ElemInstances.scala

```diff
@@ -8,7 +8,11 @@ import scala.util.control.NonFatal
 import scala.xml._
 import scalaz.concurrent.Task
 
+import javax.xml.parsers.SAXParserFactory
+
 trait ElemInstances {
+  private val spf = SAXParserFactory.newInstance()
```

Comment:
That's a good question. Creating the sax parser factory actually reads the configuration to decide what parser to use. I guess it is not strictly pure though the configuration probably never changes after you start the JVM

Any ideas on how to isolate this?